board/boards/tres.h

```diff
@@ -4,8 +4,14 @@
 
 bool tres_ir_enabled;
 bool tres_fan_enabled;
+uint8_t tres_fan_cooldown = 0U;
 void tres_update_fan_ir_power(void) {
-  red_chiplet_set_fan_or_usb_load_switch(tres_ir_enabled || tres_fan_enabled);
+  if (tres_fan_enabled) {
```

Comment:
seems a bit messy to have fan logic in so many places. if we start seeing stalls on these fans, will we have to account for this cooldown in the fan driver now?

Reply:
Agree that it feels a bit messy now, will see if there's a good way to clean it up.
Don't expect to see any issues with these fans, so don't think we need any of the stall logic for the tres

Reply:
Moved the logic inside the fan driver, and also disabled stall recovery for all boards except the dos.